soroban-sdk/src/env.rs

```diff
@@ -49,8 +50,17 @@ pub use internal::TryFromVal;
 pub use internal::TryIntoVal;
 pub use internal::Val;
 
-pub type EnvVal = internal::EnvVal<Env, RawVal>;
-pub type EnvObj = internal::EnvVal<Env, Object>;
+#[derive(Clone)]
+pub struct EnvVal {
+    pub env: Env,
+    pub val: RawVal,
+}
+
+#[derive(Clone)]
+pub struct EnvObj {
+    pub env: Env,
+    pub obj: Object,
+}
```

Comment:
I think we can get rid of these types, at least that's what I imagined we'd do, and just put Env and Object inside each types like Bytes, Vec, etc. This is fine too though.

Reply:
I have removed both of them.